test/e2e/projects-settings.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from '@playwright/test';
+import { test } from './utils/fixtures';
+import { ProjectSettingsPage } from './pages/project-settings.page';
+import { initTestProject, addUserToProject } from './utils/testSetup';
+import { ProjectsPage } from './pages/projects.page';
+
+
+export type Project = {
+  name: string,
+  code: string,
+  id: string
+}
+
+test.describe('E2E Project Settings app', () => {
+  let projectSettingsPageManager: ProjectSettingsPage;
+  const projects: Project[] = [
+    {
+      name: 'projects_settings_spec_ts Project 01',
+      code: 'p01_projects_settings_spec_ts__project_01',
+      id: ''
+    },
+    {
+      name: 'projects_settings_spec_ts Project 02',
+      code: 'p02_projects_settings_spec_ts__project_02',
+      id: ''
+    },
+    {
+      name: 'projects_settings_spec_ts Project 03',
+      code: 'p03_projects_settings_spec_ts__project_03',
+      id: ''
+    },
+  ];
+  const project4: Project = {
+    name: 'projects_settings_spec_ts Project 04',
+    code: 'p04_projects_settings_spec_ts__project_04',
+    id: ''
+  };
+
+  test.beforeAll(async ({ request, admin, member, manager, managerTab }) => {
+    projectSettingsPageManager = new ProjectSettingsPage(managerTab);
+
+    for (const project of projects) {
+      const projectId = await initTestProject(request, project.code, project.name, admin.username, [member.username]);
+      project.id = projectId;
+    }
+    await addUserToProject(request, projects[0].code, manager.username, 'manager');
+    project4.id = await initTestProject(request, project4.code, project4.name, manager.username, []);
+  });
+
+
+  // test if can change project name
```

Comment:
Should we also test whether one can change the name of a project?

Reply:
I think testing that you can change the name of the project would be nice, but not required for this PR.